src/jsutils/instanceOf.ts

```diff
@@ -1,5 +1,14 @@
 import { inspect } from './inspect';
 
+/* c8 ignore next 4 */
+const isProduction =
+  // eslint-disable-next-line no-undef
+  typeof process !== 'undefined' &&
+  // eslint-disable-next-line no-undef
+  process.env &&
+  // eslint-disable-next-line no-undef
+  process.env.NODE_ENV === 'production';
+
```

Comment:
```suggestion
/* c8 ignore next 4 */
const isDev =
  // eslint-disable-next-line no-undef
  typeof process !== 'undefined' &&
  // eslint-disable-next-line no-undef
  process.env &&
  // eslint-disable-next-line no-undef
  process.env.NODE_ENV === 'development';

```

Reply:
+1 for the dev detection instead of prod detection here. in case of a false-negative, better to hit the prod branch in dev than the dev code in prod.

Reply:
In the ongoing discussions we concluded this is a breaking change and hence not worth it for v16

Reply:
ah, makes sense - cheers for the info and cross-reference!